pkg/ovs/openflow/ofctrl_bridge_test.go

```diff
@@ -160,3 +166,78 @@ func TestOFBridgePacketRcvd(t *testing.T) {
 		}
 	}
 }
+
+func TestOFMeterStats(t *testing.T) {
+	// Reset OVS metrics (Prometheus) and reinitialize them to test.
+	legacyregistry.Reset()
+	metrics.InitializeOVSMetrics()
+
+	b := NewOFBridge("test-br", GetMgmtAddress(ovsconfig.DefaultOVSRunDir, "test-br"))
+	sw := newFakeOFSwitch(b)
+	b.SetOFSwitch(sw)
+
+	mpMeterStatsReply :=
+		openflow15.NewMpReply(openflow15.MultipartType_MeterStats)
+	meterStats1 := openflow15.NewMeterStats(1)
+	stats1 := openflow15.NewMeterBandStats()
+	stats1.PacketBandCount = 100
+	stats1.ByteBandCount = 200
+	meterStats1.AddBandStats(*stats1)
+	meterStats2 := openflow15.NewMeterStats(2)
+	stats2 := openflow15.NewMeterBandStats()
+	stats2.PacketBandCount = 0
+	stats2.ByteBandCount = 0
+	meterStats2.AddBandStats(*stats2)
+	// Unexpected case
+	meterStats3 := openflow15.NewMeterStats(3)
+	stats3 := openflow15.NewMeterBandStats()
+	stats3.PacketBandCount = 300
+	stats3.ByteBandCount = 400
+	meterStats3.AddBandStats(*stats3)
+	mpMeterStatsReply.Body = append(mpMeterStatsReply.Body, meterStats1)
+	mpMeterStatsReply.Body = append(mpMeterStatsReply.Body, meterStats2)
+	mpMeterStatsReply.Body = append(mpMeterStatsReply.Body, meterStats3)
+	handleMeterStatsReply := func(packetCounts map[int]int64) {
+		for k, v := range packetCounts {
+			if k == 1 {
+				metrics.OVSMeterPacketCount.WithLabelValues("PacketInMeterNetworkPolicy").Set(float64(v))
+			} else if k == 2 {
+				metrics.OVSMeterPacketCount.WithLabelValues("PacketInMeterTraceflow").Set(float64(v))
+			} else {
+				klog.InfoS("Received unexpected meterId", "meterId", k)
+				continue
+			}
+		}
+	}
+	expectedOVSMeterPacketCount := `
+	# HELP antrea_agent_ovs_meter_packet_count [ALPHA] Number of packets dropped by OVS meter. The value is more than 0 when the packets exceed the rate-limit.
+	# TYPE antrea_agent_ovs_meter_packet_count gauge
+   antrea_agent_ovs_meter_packet_count{meter_id="PacketInMeterNetworkPolicy"} 100
+   antrea_agent_ovs_meter_packet_count{meter_id="PacketInMeterTraceflow"} 0
+	`
+	// Timeout to wait for MeterStats reply
+	b.GetMeterStats(handleMeterStatsReply)
+	b.MultipartReply(sw, mpMeterStatsReply)
+	err := wait.PollImmediate(time.Second, 5*time.Second, func() (done bool, err error) {
+		if err = testutil.GatherAndCompare(legacyregistry.DefaultGatherer, strings.NewReader(expectedOVSMeterPacketCount), "antrea_agent_ovs_meter_packet_count"); err != nil {
+			return false, nil
+		}
+		return true, nil
```

Comment:
It's too long, you may refine these into:
```
err = testutil.GatherAndCompare(legacyregistry.DefaultGatherer, strings.NewReader(expectedOVSMeterPacketCount), "antrea_agent_ovs_meter_packet_count")
return err == nil, nil
```

Reply:
Got it, have updated.